users/migrations/0006_merge_20260119_1250.py

```
@@ -0,0 +1,14 @@
+# Generated by Django 5.2.7 on 2026-01-19 12:50
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('users', '0005_add_disable_activity_logging'),
+        ('users', '0004_alter_user_location_country'),
+    ]
```

Ruff (0.14.14)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)